File: src/codes/Users_Manual/stm/gain_2.cpp
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
auto stm =
    autd3::GainSTM(1.0f * autd3::Hz, {autd3::gain::Null(), autd3::gain::Null()})
        .with_mode(autd3::GainSTMMode::PhaseFull);
//~return 0; }

File: src/codes/Users_Manual/stm/gain_2.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let stm = GainSTM::new(1.0 * Hz, [Null::new(), Null::new()])?
    .with_mode(GainSTMMode::PhaseFull);
# Ok(())
# }